### PR TITLE
sql-select - Tim Silva

### PR DESCRIPTION
**Pull request by silva-tim**

### CODE REVIEW

File: sql-select/actor-forty.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+SELECT *
+  FROM "actors"
+ WHERE "actorId" = '40';
```
Comment (RobertGardner) [inline sql-select/actor-forty.sql:3]: Only put single quotes around strings. This is a number.

File: sql-select/cheap-to-rent.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+SELECT "filmId",
+       "title"
+  FROM "films"
+ WHERE "rentalRate" < '1.00'
```
Comment (RobertGardner) [inline sql-select/cheap-to-rent.sql:4]: Only put single quotes around strings. This is a number.

File: sql-select/inactive-customers.sql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+SELECT *
+  FROM "customers"
+ WHERE "isActive" = 'false';
```
Comment (RobertGardner) [inline sql-select/inactive-customers.sql:3]: Only put single quotes around strings. This is a boolean.

File: sql-select/second-store-customers.sql [hunk 4]
```diff
@@ -0,0 +1,3 @@
+SELECT "email"
+  FROM "customers"
+ WHERE "storeId" = '2';
```
Comment (RobertGardner) [inline sql-select/second-store-customers.sql:3]: Only put single quotes around strings. This is a number.

File: sql-select/twenty-films-of-06.sql [hunk 5]
```diff
@@ -0,0 +1,6 @@
+SELECT "title",
+       "description",
+       "releaseYear"
+  FROM "films"
+ WHERE "releaseYear" = '2006'
```
Comment (RobertGardner) [inline sql-select/twenty-films-of-06.sql:5]: Only put single quotes around strings. This is a number.

**Review by RobertGardner — CHANGES_REQUESTED** [review]
Postgres converts the strings for you, but it's bad practice to use quotes where not needed. So please correct.

File: sql-select/twenty-films-of-06.sql (hunk 5 above)
Comment (silva-tim) [reply sql-select/twenty-films-of-06.sql]: Fixed

**Review by RobertGardner — APPROVED** [review]

**Pull request merged**